CMakeLists.txt

```diff
@@ -52,14 +53,19 @@ add_subdirectory("log4tango")
 add_subdirectory("cppapi")
 
 if(BUILD_TESTING)
+  if(BUILD_SHARED_LIBS)
     add_subdirectory("cpp_test_suite")
+  else()
+    message(WARNING "Not building the tests, because that is only possible when BUILD_SHARED_LIBS is ON")
+    SET(BUILD_TESTING OFF)
```

Comment:
What is the reason for skipping tests in this case? Why can't we just link the tests statically with libtango.a?

Reply:
@mliszcz It does not work out of the box, that is the only reason. Should I get it working?

Reply:
@t-b thanks for clarification. I was just wondering why it is not possible. No need to get it working IMO. I'd just change the warning message from "it is only possible when" to "it is currently supported only when", but I do not have a strong opinion here, it's up to you.

Reply:
Good point. I'll reword when rerolling.